app/src/main/java/rick_and_morty/ui/characters/composables/CharacterList.kt

```diff
@@ -32,7 +31,7 @@ fun CharacterList(
                 items(characters.characterResults) { characters ->
```

Comment:
NIT: `characters` should be `character` since you are dealing with a single item here.

Reply:
done